duckduckgo/Android
# Feature/remove closed experiments

Task/Issue URL: https://app.asana.com/0/72649045549333/804357446611984
Tech Design URL: 
CC: 

**Description**:
Removing variants which are no longer active.

**Steps to test this PR**:
1. Migrate from existing active variant; verify you are migrated to default `""` variant key (filter logcat for `user will now use default variant`)
1. Ensure none of the experimental features are visible to you
    1. home screen call to actions (neither of them)
    1. timed banner above `WebView` (would show only after `20s` on DEBUG builds)


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
